src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -169,148 +166,111 @@ Nethermind also has [detailed documentation](https://docs.nethermind.io), strong
+- Relies on the security of the consensus mechanism.
+- Synchronization takes only a few hours.
 
-An easy way to run your own node is using 'plug and play' boxes like [DAppNode](https://dappnode.io/). It provides hardware for running clients and applications that depend on them with a simple user interface.
+#### Light sync
 
-### Requirements {#requirements}
+Light client mode downloads all block headers, block data, and verifies some randomly. Only syncs tip of the chain from the trusted checkpoint.
 
-Before installing any client, please ensure your computer has enough resources to run it. Minimum and recommended requirements can be found below, however the key part is the disk space. Syncing the Ethereum blockchain is very input/output intensive. It is best to have a solid-state drive (SSD). To run an Ethereum client on HDD, you will need at least 8GB of RAM to use as a cache.
+- Gets only the latest state while relying on trust in developers and consensus mechanism.
+- Client ready to use with current network state in a few minutes.
 
-#### Minimum requirements {#recommended-specifications}
+[More on Light clients](https://www.parity.io/blog/what-is-a-light-client/)
 
-- CPU with 2+ cores
-- 4 GB RAM minimum with an SSD, 8 GB+ if you have an HDD
-- 8 MBit/s bandwidth
+#### Snap sync
 
-#### Recommended specifications {#recommended-specifications}
+The latest approach to syncing the client, pioneered by Geth. Using dynamic snapshots served by peers retrieves all the account and storage data without downloading intermediate trie nodes and then reconstructs the Merkle trie locally.
 
-- Fast CPU with 4+ cores
-- 16 GB+ RAM
-- Fast SSD with at least 500 GB free space
-- 25+ MBit/s bandwidth
+- Fastest sync strategy, currently default in Geth and Nethermind 
+- Saves a lot of disk usage and network bandwidth without sacrificing security.
 
-The sync mode you choose will affect space requirements but we've estimated the disk space you'll need for each client below.
+[More on Snap](https://github.com/ethereum/devp2p/blob/master/caps/snap.md)
 
-| Client       | Disk size (fast sync) | Disk size (full archive) |
-| ------------ | --------------------- | ------------------------ |
-| Geth         | 400GB+                | 6TB+                     |
-| OpenEthereum | 280GB+                | 6TB+                     |
-| Nethermind   | 200GB+                | 5TB+                     |
-| Besu         | 750GB+                | 5TB+                     |
-| Erigon       | N/A                   | 1TB+                     |
 
-- Note: Erigon does not Fast Sync, but Full Pruning is possible (~500GB)
+#### Beam sync
```

Comment:
Is this still supported in Nethermind? If not, I'd remove it as it was never production ready IIRC